crates/fluvio-protocol/Cargo.toml

```diff
@@ -36,6 +36,7 @@ tokio-util = { version = "0.7.0", features = [
     "compat",
 ], optional = true }
 semver = { version = "1.0.0", optional = true }
+serde = { version = "1.0.147", features = ['derive'], optional = true }
```

Comment:
Do we still need this? I dont' see ref in the crate anymore

Reply:
Thanks! Addressed on: [a63b4b2](https://github.com/infinyon/fluvio/pull/2738/commits/a63b4b25a26acd1d627d7442798ad30fcfe48bb7)